public/translations/en-US.json

```diff
@@ -104,6 +104,10 @@
     "invalidSymbol": "Invalid symbol.",
     "unknown": "Uoh. Something wrong happened.",
     "accountNotFound": "Account not found. Are you sure you created your account?",
+    "accountDoesNotCorrespond": "This private key does not correspond to logged in account. Please logout to use a different account.",
```

Comment:
If we ever got to this state, we cannot do nothing about it, the session is doomed, maybe we can show a modal displaying this message and force a logout?

Reply:
Honestly, I don't even think it's possible to get to this state, but it was there before and I couldn't figure out how to test it, so I just kept it, but yes, we can do something like that.

Also, does it still make sense to get multiple `account_name`s for the same public key/user (there was a `LoginWithMultipleAccounts` state before that I removed, because it wasn't being generated)? Can we be sure that the correct account (if there is one) will always be at the first position in the list?

Reply:
Yes, this case is actually impossible.
For it to happen, the user must be logged in (have their info on localStorage), but this port is only called from the `Login` page (which means the user is not logged in). But, **if** this ever happens, I thinks it's safe to just ignore it: it would be the same as the user logging out and logging back in, with a different account (we can make sure all of the data a logged in user can have in localStorage is deleted every time on login, just to be safe)